src/api/productsApi.ts

```diff
@@ -20,13 +20,24 @@ export const productsApi = createApi({
   endpoints: (build) => ({
     getProducts: build.query<
       ProductsResponse,
-      { categoryId?: string; limit?: number; offset?: number }
+      { categoryId?: string; sortOption?: string; limit?: number; offset?: number }
     >({
-      query: ({ categoryId = 'root', limit = PRODUCTS_LIMIT, offset = OFFSET }) => {
-        if (categoryId === 'root' || !categoryId) {
-          return `/product-projections?limit=${limit}&offset=${offset}`;
+      query: ({
+        categoryId = 'root',
+        sortOption = '',
+        limit = PRODUCTS_LIMIT,
+        offset = OFFSET,
+      }) => {
+        const searchParameters = [];
```

Comment:
I think we can use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) here.